R/selections.R

```diff
@@ -127,12 +127,12 @@ eval_select_recipes <- function(quos, data, info, ..., allow_rename = FALSE) {
   # used for both the training and test set and their positions
   # may have changed. If renaming is allowed, add the new names.
   out <- names(data)[sel]
-  if (allow_rename) names(out) <- names(sel)
+  names(out) <- names(sel)
```

Comment:
It might be cleaner to do something like

```r
names <- names(sel)
```

then in the if statement below:

```r
if (!allow_rename & !identical(out, names)) {
```

then after the if statement

```r
names(out) <- names
```

Reply:
I like that. I had in mind to write it so that we could just delete the block with the if statement when it's time for that - but for the time being this is nicer!